src/UI/SearchAndPreferences.jsx

```diff
@@ -49,7 +50,7 @@ const SearchAndPreferences = () => {
           }}
           placeholder="Search"
           type={'text'}
-          color="secondary"
+          className="bg-transparent"
```

Comment:
If you're having hard time customizing MUI components, take a look at [this](https://mui.com/material-ui/customization/theme-components/#theme-style-overrides)

TLDR; you can create theme objects, one for light one for dark theme, and configure what ever colors/components you wish there

Then you just toggle the themes together with the dark/light